### DISCUSSION TITLE
Get descriptor without training

### DISCUSSION THREAD

**Discussion by braxtonowens** [deepmodeling/deepmd-kit#1688] [Q&A]
Is there any way to retrieve the descriptor without also training? 

Ideally, with some input, like an ASE object or LAMMPS dump I could just retrieve the descriptor? 

Essentially, something like the Dscribe python library? https://singroup.github.io/dscribe/latest/

**Comment by njzjz** [comment]
DeePMD-kit's descriptors are trainable, unlike others.

**Reply by braxtonowens** [reply]
I don't quite understand your response. What do you mean by trainable?

I just want the described representation of my data. Is there a way I can do this?

**Reply by njzjz** [reply]
The parameters of the descriptor are trainable. The initial parameters are just random values. So there is no way to retrieve the descriptor without training. You need to train a model firstly, and then infer the descriptor.
